shared/reducers/favorite.js

```diff
@@ -19,6 +20,12 @@ export default function (state: State = initialState, action: FavoriteAction): S
         ...state,
         folders: action.payload && action.payload.folders
       }
+    case logoutDone: {
```

Comment:
That seems fine to me

On Friday, March 4, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/reducers/favorite.js
> <https://github.com/keybase/client/pull/2228#discussion_r55111397>:
>
> > @@ -19,6 +20,12 @@ export default function (state: State = initialState, action: FavoriteAction): S
> >          ...state,
> >          folders: action.payload && action.payload.folders
> >        }
> > +    case logoutDone: {
>
> What if we just reset the store to it's initial state?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2228/files#r55111397>.
>